src/components/Header/DesktopNav.client.tsx

```diff
@@ -52,7 +52,10 @@ export const DesktopNav = ({
 
           return (
             <NavigationMenuItem key={label} value={label}>
-              <NavigationMenuTrigger className="data-[state=open]:text-header-foreground-highlight font-medium">
+              <NavigationMenuTrigger
+                onClick={(e) => e.preventDefault()}
```

Comment:
This disables accessibility via clicking space bar or enter when this element is focused. How about this instead:

```suggestion
             onClick={(e) => {
                  // maintains accessibility via clicking space bar or enter with this trigger focused
                  // but disables mouse clicks or touch events
                  if ('pointerType' in e.nativeEvent && e.nativeEvent.pointerType !== '') {
                    e.preventDefault()
                  }
                }}
```